B2MML-ProcessEvents.xsd

```diff
@@ -29,7 +29,8 @@
   <xsd:include schemaLocation="B2MML-WorkCapability.xsd" />
   <xsd:include schemaLocation="B2MML-WorkPerformance.xsd" />
   <xsd:include schemaLocation="B2MML-WorkSchedule.xsd" />
-  
+  <xsd:include schemaLocation="B2MML-ErrorMessage.xsd"/>
+  <xsd:include schemaLocation="B2MML-OperationsTest.xsd"/>

```

Comment:
Issue #11 to bring in the new OperationsTest.xsd but this, and the old TestSpecification.xsd include statement may still be removed after some further investigation.
For now, I am adding this line, but looks like the ProcessEvents.xsd will operate fine without them, via extensions